internal/config/config.go

```diff
@@ -594,6 +598,7 @@ var (
 		FailoverStoragesCheckSize:            true,
 		DisablePartialRestore:                true,
 		ForceWalDetal:                        true,
+		PgAppName:                            true,
```

Comment:
this line is optional - all GP inherits all PG settings:

https://github.com/wal-g/wal-g/blob/master/internal/configure.go#L478-L480